src/App/routes.tsx

```diff
@@ -706,6 +706,79 @@ const allSurgeAlerts = customWrapRoute({
     },
 });
 
+const catalogueEmergency = customWrapRoute({
+    path: 'emergency',
+    component: {
+        render: () => import('#views/CatalogueEmergency'),
+        props: {},
+    },
+    parent: surgeCatalogue,
+    wrapperComponent: Auth,
+    context: {
+        title: 'Emergency Catalogue',
+        visibility: 'anything',
+    },
+});
+
+const catalogueEmergencyIndex = customWrapRoute({
+    index: true,
+    component: {
+        render: () => import('#views/CatalogueEmergencyIndex'),
+        props: {},
+    },
+    parent: catalogueEmergency,
+    wrapperComponent: Auth,
+    context: {
+        title: 'Emergency Catalogue',
+        visibility: 'anything',
+    },
+});
+
+const assessmentCell = customWrapRoute({
+    path: 'assessment-cell',
+    component: {
+        render: () => import('#views/AssessmentCell'),
+        props: {},
+    },
+    parent: catalogueEmergency,
+    wrapperComponent: Auth,
+    context: {
+        title: 'Assessment Cell',
+        visibility: 'anything',
+    },
+});
+
+const catalogueService = customWrapRoute({
+    path: 'service',
+    component: {
+        render: () => import('#views/CatalogueService'),
+        props: {},
+    },
+    parent: surgeCatalogue,
+    wrapperComponent: Auth,
+    context: {
+        title: 'Surge Services Catalogue',
+        visibility: 'anything',
+    },
+});
+
+const catalogueIndex = customWrapRoute({
+    parent: surgeCatalogue,
+    index: true,
+    component: {
+        eagerLoad: true,
+        render: Navigate,
```

Comment:
@frozenhelium do we need to navigate or just display catalogueService page in the index route

Reply:
We probably need to navigate